src/xdsl/__init__.py

```diff
@@ -0,0 +1,38 @@
+_mlir_module = None
+_REQUIRED_MLIR_MODULES = ['ir']
```

Comment:
For my understanding: This are bascially all the modules that xdsl needs for the MLIRConverter? Since the newly loaded module needs to have those imported as well.

Reply:
Yes, exactly. If the user forgets to import one of those (e.g., if she only does `import mlir_iterators` without `import mlir_iterators.ir`), an error will be raised right away.